core/src/main/java/org/bitcoinj/crypto/MnemonicCode.java

```diff
@@ -53,6 +57,14 @@
     private static final String BIP39_ENGLISH_SHA256 = "ad90bf3beb7b0eb7e5acd74727dc0da96e0a280a258354e7293fb7e211ac03db";
     private static final int WORD_LIST_SIZE = 2048;
     private static final int PBKDF2_ROUNDS = 2048;
+    private static final Normalizer.Form NORMALIZE_FORM = Normalizer.Form.NFKD;
+    private static final Function<String, String> NORMALIZE_FUNCTION = new Function<String, String>() {
+        @NullableDecl
```

Comment:
I'm not sure about the differences of @NullableDecl to @Nullable, but so far we've used the latter.